tests/unit/moduleapi/keyspecs.tcl

```diff
@@ -4,53 +4,38 @@ start_server {tags {"modules"}} {
     r module load $testmodule
 
     test "Module key specs: Legacy" {
-        set reply [r command info kspec.legacy]
+        set reply [lindex [r command info kspec.legacy] 0]
         # Verify (first, last, step)
-        assert_equal [lindex [lindex $reply 0] 3] 1
-        assert_equal [lindex [lindex $reply 0] 4] 2
-        assert_equal [lindex [lindex $reply 0] 5] 1
-        # create a dict for easy lookup
-        unset -nocomplain mydict
-        foreach {k v} [lindex [lindex $reply 0] 7] {
-            dict append mydict $k $v
-        }
```

Comment:
@guybe7 FYI, simpler way to turn a list into a dict:
`[dict create {*}$list]`

Reply:
thanks, i copied it from somewhere else, maybe we should use the one-liner there too